interfaces.go

```diff
@@ -0,0 +1,171 @@
+package linodego
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/linode/linodego/internal/parseabletime"
+)
+
+type LinodeInterface struct {
+	ID           int                    `json:"id"`
+	MACAddress   string                 `json:"mac_address"`
+	Created      *time.Time             `json:"-"`
+	Updated      *time.Time             `json:"-"`
+	DefaultRoute InterfacesDefaultRoute `json:"default_route"`
+	Version      int                    `json:"version"`
+	VPC          InterfacesVPC          `json:"vpc"`
+	Public       InterfacesPublic       `json:"public"`
+	VLAN         InterfacesVLAN         `json:"vlan"`
+}
+
+type InterfacesDefaultRoute struct {
+	IPv4 bool `json:"ipv4,omitempty"`
+	IPv6 bool `json:"ipv6,omitempty"`
```

Comment:
```suggestion
	IPv4 *bool `json:"ipv4,omitempty"`
	IPv6 *bool `json:"ipv6,omitempty"`
```
Let's make them pointers to avoid confusions